app/views/pages/score_information.html.erb

```diff
@@ -0,0 +1,39 @@
+<div class="container">
+  <div class="row-container px-5 py-3">
+    <div class="py-3">
+      <div class="text-center py-3">
+        <h1>Computing the ECO-Score</h1>
+        <div class="py-3">
+          <p>Some fancy text about the ECO-Score</p>
+        </div>
+      </div>
+      <div class="">
+        <div class="card-product">
+          <div class="card-product-infos">
```

Comment:
If possible, it would be nice to have a different layout to the materials index page, i guess even if you're using card-product code, it will look different in the end?